UnityAds/src/main/java/com/applovin/mediation/adapters/UnityAdsMediationAdapter.java

```diff
@@ -73,13 +76,11 @@ public void initialize(final MaxAdapterInitializationParameters parameters, fina
             log( "Initializing UnityAds SDK with game id: " + gameId + "..." );
             initializationStatus = InitializationStatus.INITIALIZING;
 
-            if ( serverParameters.getBoolean( KEY_SET_MEDIATION_IDENTIFIER ) )
-            {
-                MediationMetaData mediationMetaData = new MediationMetaData( context );
-                mediationMetaData.setName( UnityAdsMediationAdapter.mediationTag() );
-                mediationMetaData.setVersion( AppLovinSdk.VERSION );
-                mediationMetaData.commit();
-            }
+            MediationMetaData mediationMetaData = new MediationMetaData( context );
+            mediationMetaData.setName( MEDIATION_TAG );
+            mediationMetaData.setVersion( UnityAds.getVersion() );
+            mediationMetaData.set( KEY_ADAPTER_VERSION, AppLovinSdk.VERSION );
```

Comment:
Want to double-check this is intended - the "adapter version" is actually pointing to our SDK's version, not the actual adapter's verison.